detox/test/e2e/init.js

```diff
@@ -22,7 +22,12 @@ beforeAll(async () => {
 }, 300000);
 
 beforeEach(async () => {
-  await adapter.beforeEach();
+  try {
+    await adapter.beforeEach();
+  } catch (err) {
+    await detox.cleanup();
```

Comment:
This is a trick for `jest-jasmine`, to set to `Detox.none`, when tests run forward while `detox.init()` has not ended well. I could have put that `detox.cleanup()` inside `adapter.beforeEach()`, but my gut feeling tells me that better to leave it exposed to the user. Moreover, this trick is specific to jest-jasmine only.